pt-br/pyqt-pt.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt.py
```

Comment:
Please change the filename according to other formats. You can check folders for filenames.  The desired filename is: 
learnpyqt-pt.py